docs/develop/api-reference/http-rest-api.md

```diff
@@ -0,0 +1,212 @@
+
+## Access
+
+For access to the HTTP API, use one of the following endpoints in the region your caches are in and you have a Momento auth token for.
+
+|   | AWS Region     	| Endpoints                                        	|
+|---|----------------|------------------------------------------------------|
+| 1 | us-west-2  	| api.cache.cell-4-us-west-2-1.prod.a.momentohq.com	|
+| 2 | us-east-1  	| api.cache.cell-us-east-1-1.prod.a.momentohq.com  	|
+| 3 | ap-northeast-1 | api.cache.cell-ap-northeast-1-1.prod.a.momentohq.com |
+| 4 | eu-west-1  	| api.cache.cell-1-eu-west-1-1.prod.a.momentohq.com	|
+| 5 | ap-south-1 	| api.cache.cell-1-ap-south-1-1.prod.a.momentohq.com   |
+
+If you do not see a Region you require, let’s discuss. Please contact [Support](mailto:support@momentohq.com).
+
+## Get Method
+Gets a scalar value from a cache.
+### Request
+
+- Path: /cache/{cacheName}
+- HTTP Method: GET
+
+#### Path Parameters
+
+| Parameter&nbsp;name | Required?           | Type            | Description |
+| -------------- |---------------------|-----------------|----------------------------|
+| cacheName | yes | URL-safe string | The name of the cache to operate on. |
+
+#### Query Parameters
+
+| Parameter&nbsp;name | Required? | Type                                         | Description           |
+|---------------|-----------|------|------------------|
+| key           | no        | URL-safe&nbsp;string                         | The key to retrieve from the cache. Must specify either this parameter or `key_base64`.|
+| key_base64    | no        | Base64&nbsp;URL-encoded&nbsp;binary&nbsp;key | The key to retrieve from the cache. Must specify either this parameter or `key_base64`.|
+| token         | no        | URL-safe string                              | The Momento auth token to be used for authentication/authorization of the request. Must be provided either here or as the `authorization` header in the HTTP call. |
+
+### Response
+#### Hit
+*Status Code: 200 OK*
+
+- Content-Type: octet-stream
+- Body: The binary value stored at the specified key.
+
+#### Miss
+*Status Code: 404 Not Found*
+
+- "Key Not Found" indicates that the key was not present in the cache, while "Cache Not Found" indicates that the specified cache does not exist.
+
+#### Error
+
+*Status Code: 400 Bad Request*
+- This error type typically indicates that the request was incorrectly specified. See the body of the message for further details.
+
+*Status Code: 401 Unauthorized*
+
+- This error type typically indicates that there was an issue with authentication/authorization. See the body of the message for further details.
```

Comment:
Per feedback from bug bash, we've fixed some error mapping and this will need to be broken into two parts for all three APIs:

- 401 Unauthorized - issues with authentication.
- 403 Forbidden - issues with authorization/FGAC.

(I know, the HTTP status names are confusing compared to meaning, but that's the HTTP spec...)

Reply:
Fixed and pushed a commit.